### PR TITLE
OCM-10248 | test: automated cases id:75052

### PR DESCRIPTION
**Pull request by jfrazierRH**

### CODE REVIEW

**Comment by jfrazierRH** [comment]
`jfrazier@jfrazier-mac  ~/workspace/rosa   jf_75052 ±  ginkgo run -focus 75052 tests/e2e
Ginkgo detected a version mismatch between the Ginkgo CLI and the version of Ginkgo imported by your packages:
  Ginkgo CLI Version:
    2.11.0
  Mismatched package versions found:
    2.17.1 used by e2e

  Ginkgo will continue to attempt to run but you may see errors (including flag
  parsing errors) and should either update your go.mod or your version of the
  Ginkgo CLI to match.

  To install the matching version of the CLI run
    go install github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file.  Alternatively you can use
    go run github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file to invoke the matching version of the
  Ginkgo CLI.

  If you are attempting to test multiple packages that each have a different
  version of the Ginkgo library with a single Ginkgo CLI that is currently
  unsupported.
  
Running Suite: ROSA CLI e2e tests suite - /Users/jfrazier/workspace/rosa/tests/e2e
==================================================================================
Random Seed: 1723050476

Will run 1 of 171 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 171 Specs in 3.025 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 170 Skipped
PASS

Ginkgo ran 1 suite in 8.324353667s
Test Suite Passed`

File: tests/e2e/test_rosacli_ingress.go [hunk 1]
```diff
@@ -161,6 +161,23 @@ var _ = Describe("Edit default ingress",
...
+			func() {
+				By("Get default ingress id with no clusterID provided")
+				emptyClusterID := ""
+				output, err := ingressService.ListIngress(emptyClusterID)
+				Expect(err).To(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring(
+					"ERR: Cluster name, identifier or external identifier '' " +
+						"isn't valid: it must contain only letters, digits, dashes and underscore"))
+
+				By("Get default ingress id with no clusterID provided")
```
Comment (yingzhanredhat) [inline tests/e2e/test_rosacli_ingress.go:175]: Just one little advice: I think the title should "Get cluster ingress with invalid/non-exited id"

**Review by radtriste — APPROVED** [review]
/lgtm

**Pull request merged**